### PR TITLE
Added Spanish Translation

### PR DESCRIPTION
**Pull request by GasInfinity**
I added spanish language file

### CODE REVIEW

**Comment by Sturmlilie** [comment]
Thank you very much, I'll have someone proof read it :)

**Comment by Sturmlilie** [comment]
Some feedback from discord:
```
Mr Louwind: code_book should be "Libro de Código" o "Manual de Código"
Mr Louwind: Also, it's Recibir not Recivir
Mr Louwind: And Módulo not Modulo
```

**Comment by GasInfinity** [comment]
Oh, sorry, I will change it

File: src/main/resources/assets/tis3d/lang/es_es.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    "item.tis3d.module_terminal": "Módulo de Terminal",
+    "item.tis3d.module_timer": "Módulo de Temporizador",
+    "item.tis3d.prism": "Prisma",
+
+    "block.tis3d.casing": "Cubierta",
+    "block.tis3d.controller": "Controlador",
+
+    "tis3d.compiler.error": "Error de Compilacion @%s:%s-%s: ",
+    "tis3d.compiler.invalid_format": "Símbolo inesperado",
+    "tis3d.compiler.invalid_instruction": "Opcode Invalido",
```
Comment (FlashyReese) [inline src/main/resources/assets/tis3d/lang/es_es.json:31]:
```suggestion
    "tis3d.compiler.invalid_instruction": "Opcode inválido",
```
or
```suggestion
    "tis3d.compiler.invalid_instruction": "Código de operación inválido",
```

File: src/main/resources/assets/tis3d/lang/es_es.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    "tis3d.compiler.label_not_found": "Etiqueta no encontrada",
+    "tis3d.compiler.label_duplicate": "La etiqueta ya esta definida",
```
Comment (FlashyReese) [inline src/main/resources/assets/tis3d/lang/es_es.json:33]:
```suggestion
    "tis3d.compiler.label_duplicate": "La etiqueta ya está definida",
```

File: src/main/resources/assets/tis3d/lang/es_es.json [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    "tis3d.compiler.parameter_invalid": "Registro Invalido",
```
Comment (FlashyReese) [inline src/main/resources/assets/tis3d/lang/es_es.json:34]:
```suggestion
    "tis3d.compiler.parameter_invalid": "Registro inválido",
```

File: src/main/resources/assets/tis3d/lang/es_es.json [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    "tis3d.compiler.parameter_overflow": "Muchos Operandos",
```
Comment (FlashyReese) [inline src/main/resources/assets/tis3d/lang/es_es.json:35]:
```suggestion
    "tis3d.compiler.parameter_overflow": "Demasiados operandos",
```

File: src/main/resources/assets/tis3d/lang/es_es.json [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    "tis3d.compiler.parameter_underflow": "Faltan Operandos",
```
Comment (FlashyReese) [inline src/main/resources/assets/tis3d/lang/es_es.json:36]:
```suggestion
    "tis3d.compiler.parameter_underflow": "Faltan operandos",
```
or
```suggestion
    "tis3d.compiler.parameter_underflow": "Operando faltante",
```

File: src/main/resources/assets/tis3d/lang/es_es.json [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+    "tis3d.compiler.too_many_columns": "Linea muy larga",
```
Comment (FlashyReese) [inline src/main/resources/assets/tis3d/lang/es_es.json:37]:
```suggestion
    "tis3d.compiler.too_many_columns": "Línea demasiado larga",
```

File: src/main/resources/assets/tis3d/lang/es_es.json [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+    "tis3d.compiler.too_many_lines": "Demasiadas lineas",
```
Comment (FlashyReese) [inline src/main/resources/assets/tis3d/lang/es_es.json:38]:
```suggestion
    "tis3d.compiler.too_many_lines": "Demasiadas líneas",
```

File: src/main/resources/assets/tis3d/lang/es_es.json [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+
+    "tis3d.controller.status.incomplete": "El Escaneo ha fallado en completarse",
```
Comment (FlashyReese) [inline src/main/resources/assets/tis3d/lang/es_es.json:40]:
```suggestion
    "tis3d.controller.status.incomplete": "No se pudo completar el escaneo",
```

File: src/main/resources/assets/tis3d/lang/es_es.json [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+    "tis3d.controller.status.multiple_controllers": "Multiples Controladores",
```
Comment (FlashyReese) [inline src/main/resources/assets/tis3d/lang/es_es.json:41]:
```suggestion
    "tis3d.controller.status.multiple_controllers": "Varios controladores",
```
or
```suggestion
    "tis3d.controller.status.multiple_controllers": "Múltiples controladores",
```

File: src/main/resources/assets/tis3d/lang/es_es.json [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+    "tis3d.controller.status.too_complex": "Muy Complejo",
```
Comment (FlashyReese) [inline src/main/resources/assets/tis3d/lang/es_es.json:42]:
```suggestion
    "tis3d.controller.status.too_complex": "Demasiado complejo",
```

File: src/main/resources/assets/tis3d/lang/es_es.json [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+
+    "tis3d.manual.home": "Casa",
+    "tis3d.manual.blocks": "Bloques",
+    "tis3d.manual.items": "Items",
```
Comment (FlashyReese) [inline src/main/resources/assets/tis3d/lang/es_es.json:46]:
```suggestion
    "tis3d.manual.items": "Ítems",
```

File: src/main/resources/assets/tis3d/lang/es_es.json [hunk 12]
```diff
@@ -0,0 +1,59 @@
...
+    "tis3d.manual.serial_protocols": "Protocolos Seriales",
+    "tis3d.manual.warning.missing.block": "Bloque no disponible.",
+    "tis3d.manual.warning.missing.image": "Imagen no Encontrada.",
```
Comment (FlashyReese) [inline src/main/resources/assets/tis3d/lang/es_es.json:49]:
```suggestion
    "tis3d.manual.warning.missing.image": "Imagen no encontrada.",
```

File: src/main/resources/assets/tis3d/lang/es_es.json [hunk 13]
```diff
@@ -0,0 +1,59 @@
...
+    "tis3d.manual.warning.missing.item": "Item no Disponible.",
```
Comment (FlashyReese) [inline src/main/resources/assets/tis3d/lang/es_es.json:50]:
```suggestion
    "tis3d.manual.warning.missing.item": "Ítem no disponible.",
```

File: src/main/resources/assets/tis3d/lang/es_es.json [hunk 14]
```diff
@@ -0,0 +1,59 @@
...
+    "tis3d.manual.warning.missing.tag": "Etiqueta no disponible.",
+
+    "tis3d.book_code.error_on_next_page": "Error en la siguiente pagina:",
```
Comment (FlashyReese) [inline src/main/resources/assets/tis3d/lang/es_es.json:53]:
```suggestion
    "tis3d.book_code.error_on_next_page": "Error en la página siguiente:",
```

File: src/main/resources/assets/tis3d/lang/es_es.json [hunk 15]
```diff
@@ -0,0 +1,59 @@
...
+    "tis3d.book_code.error_on_previous_page": "Error en la anterior pagina:",
```
Comment (FlashyReese) [inline src/main/resources/assets/tis3d/lang/es_es.json:54]:
```suggestion
    "tis3d.book_code.error_on_previous_page": "Error en la página anterior:",
```

File: src/main/resources/assets/tis3d/lang/es_es.json [hunk 16]
```diff
@@ -0,0 +1,59 @@
...
+    "item.tis3d.module_timer": "Módulo de Temporizador",
+    "item.tis3d.prism": "Prisma",
+
+    "block.tis3d.casing": "Cubierta",
+    "block.tis3d.controller": "Controlador",
+
+    "tis3d.compiler.error": "Error de Compilacion @%s:%s-%s: ",
+    "tis3d.compiler.invalid_format": "Símbolo inesperado",
+    "tis3d.compiler.invalid_instruction": "Opcode Invalido",
+    "tis3d.compiler.label_not_found": "Etiqueta no encontrada",
```
Comment (FlashyReese) [inline src/main/resources/assets/tis3d/lang/es_es.json:32]:
```suggestion
    "tis3d.compiler.label_not_found": "Etiqueta indefinida",
```

File: src/main/resources/assets/tis3d/lang/es_es.json [hunk 17]
```diff
@@ -0,0 +1,59 @@
...
+    "tis3d.manual.serial_protocols": "Protocolos Seriales",
+    "tis3d.manual.warning.missing.block": "Bloque no disponible.",
+    "tis3d.manual.warning.missing.image": "Imagen no Encontrada.",
+    "tis3d.manual.warning.missing.item": "Item no Disponible.",
+    "tis3d.manual.warning.missing.tag": "Etiqueta no disponible.",
+
+    "tis3d.book_code.error_on_next_page": "Error en la siguiente pagina:",
+    "tis3d.book_code.error_on_previous_page": "Error en la anterior pagina:",
+
+    "tis3d.tooltip.book_code": "§7Usa un libro en un modulo de ejecucion instalado para conseguir una copia.",
```
Comment (FlashyReese) [inline src/main/resources/assets/tis3d/lang/es_es.json:56]:
```suggestion
    "tis3d.tooltip.book_code": "§7Usa un libro sobre un módulo de ejecución instalado para obtener una copia.",

```

File: src/main/resources/assets/tis3d/lang/es_es.json [hunk 18]
```diff
@@ -0,0 +1,59 @@
...
+    "tis3d.tooltip.book_manual": "§7Usa un libro en un bloque controlador para conseguir una copia.",
```
Comment (FlashyReese) [inline src/main/resources/assets/tis3d/lang/es_es.json:57]:
```suggestion
    "tis3d.tooltip.book_manual": "§7Usa un libro sobre un bloque de controlador para obtener una copia.",
```

File: src/main/resources/assets/tis3d/lang/es_es.json [hunk 19]
```diff
@@ -0,0 +1,59 @@
...
+    "tis3d.tooltip.key": "§7Agachate mientras estas sujetando para recibir puertos en una cara de la cubierta."
```
Comment (FlashyReese) [inline src/main/resources/assets/tis3d/lang/es_es.json:58]:
```suggestion
    "tis3d.tooltip.key": "§7Agachate mientras estas sujetando para recibir puertos en una cara de la carcasa."
```

File: src/main/resources/assets/tis3d/lang/es_es.json [hunk 20]
```diff
@@ -0,0 +1,59 @@
...
+    "item.tis3d.module_read_only_memory": "Módulo de ROM",
+    "item.tis3d.module_redstone": "Módulo de Redstone",
+    "item.tis3d.module_sequencer": "Módulo Secuenciador",
+    "item.tis3d.module_serial_port": "Módulo de Puerto Serial",
+    "item.tis3d.module_stack": "Módulo de Estaqueo",
+    "item.tis3d.module_terminal": "Módulo de Terminal",
+    "item.tis3d.module_timer": "Módulo de Temporizador",
+    "item.tis3d.prism": "Prisma",
+
+    "block.tis3d.casing": "Cubierta",
```
Comment (FlashyReese) [inline src/main/resources/assets/tis3d/lang/es_es.json:26]:
```suggestion
    "block.tis3d.casing": "Carcasa",
```

File: src/main/resources/assets/tis3d/lang/es_es.json [hunk 21]
```diff
@@ -0,0 +1,59 @@
...
+    "item.tis3d.key": "Llave",
+    "item.tis3d.key_creative": "Llave Maestra",
+    "item.tis3d.module_audio": "Módulo de Audio",
+    "item.tis3d.module_bundled_redstone": "Módulo Redstone incluido",
+    "item.tis3d.module_display": "Módulo de Pantalla",
+    "item.tis3d.module_execution": "Módulo de Ejecucion",
+    "item.tis3d.module_infrared": "Módulo de Infrarrojos",
+    "item.tis3d.module_keypad": "Módulo de Teclado",
+    "item.tis3d.module_queue": "Módulo de Cola",
+    "item.tis3d.module_random": "Módulo Random",
```
Comment (FlashyReese) [inline src/main/resources/assets/tis3d/lang/es_es.json:15]: Depending on context here
```suggestion
    "item.tis3d.module_random": "Módulo Randomizador",
```

**Review by FlashyReese — CHANGES_REQUESTED** [review]
I decided to look over this as requested from the Fabric Translator's Guild
Here is the discord invite https://discord.gg/cDJVatJ
You can find on there

**Pull request closed without merging**